src/__tests__/__snapshots__/typescript.test.ts.snap

```diff
@@ -0,0 +1,331 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`TypescriptPlugin snapshot 1`] = `
+Object {
+  "IButtonProps": Object {
+    "documentation": undefined,
+    "fileName": "/Volumes/git/documentalist/src/__tests__/fakes/button.ts",
```

Comment:
filename should be excluded from the snapshot -- it differs by dev machine

Reply:
hmm that's absolutely true. i've been thinking of making `fileName` relative to root, so it would just be `src/...`, which _should be_ the same on every machine. thoughts on that approach?